packages/server/lib/util/ci_provider.js

```diff
@@ -197,6 +198,16 @@ const _providerCiParams = () => {
       'BUILDKITE_PULL_REQUEST_BASE_BRANCH',
       'BUILDKITE_RETRY_COUNT',
     ]),
+    // All available vars
```

Comment:
@trojano31 @estrada9166 on our Cloud team pointed this out:

> we need to still send the customCiBuildId in case record jobs that run it in parallel or with groups

This change would be made here: https://github.com/cypress-io/cypress-services/blob/develop/packages/util/src/ciProvider.ts#L25-L45 

Looking at the Buddy envs, it looks like `BUDDY_PIPELINE_ID ` and `BUDDY_EXECUTION_ID` would satisfy this.

Reply:
@emilyrohrbough  I assume the change has to be done by the Cloud team as I don't have permissions to view this repository? However, I've added `BUDDY_PIPELINE_ID` to the list of extracted variables